htdocs/install/mysql/migration/16.0.0-17.0.0.sql

```diff
@@ -142,3 +142,6 @@ CREATE TABLE llx_bank_extrafields
 )ENGINE=innodb;
 
 ALTER TABLE llx_bank_extrafields ADD INDEX idx_bank_extrafields (fk_object);
+
+ALTER TABLE llx_product_lot ADD COLUMN note_public text DEFAULT NULL after batch;
```

Comment:
Can you also update the file   htdocs/install/mysql/table/llx_product_lot.sql
so fields will exists for new installation too...